pod/perlvar.pod

```diff
@@ -984,14 +1012,14 @@ find uses of these problematic match variables in your code.
 X<$1> X<$2> X<$3> X<$I<digits>>
 
 Contains the subpattern from the corresponding set of capturing
-parentheses from the last successful pattern match, not counting patterns
-matched in nested blocks that have been exited already.
+parentheses from the last successful pattern match in the current
+dynamic scope. (See L</Scoping Rules of Regex Variables>.)
 
 Note there is a distinction between a capture buffer which matches
 the empty string a capture buffer which is optional. Eg, C<(x?)> and
 C<(x)?> The latter may be undef, the former not.
 
-These variables are read-only and dynamically-scoped.
```

Comment:
I feel like there isn't much benefit to remove "and dynamically-scoped" from all of these. It provides additional quick information for someone who has seen the definition before, and something to look up otherwise.

Reply:
I suppose you're removing it because the variable itself is global, it just gets set in a dynamic scope, so I'm fine with it either way.

Reply:
@grinnz: I decided to go with a compromise, and changed the wording to "and its value is dynamically scoped". (Do I need the hyphen?)

Reply:
I havent reviewed the patch fully yet (at a glance it looks fine to me) but that works for me.